ipn/ipnlocal/local.go

```diff
@@ -3342,11 +3350,7 @@ func dnsConfigForNetmap(nm *netmap.NetworkMap, peers map[tailcfg.NodeID]tailcfg.
 		//
 		// While we're already populating it, might as well size the
 		// slice appropriately.
-		dcfg.Routes[fqdn] = make([]*dnstype.Resolver, 0, len(resolvers))
-
-		for _, r := range resolvers {
-			dcfg.Routes[fqdn] = append(dcfg.Routes[fqdn], r)
-		}
+		dcfg.Routes[fqdn] = append(dcfg.Routes[fqdn], resolvers...)
```

Comment:
this shows up as `nil` vs `[]`. I don't know if that matters

Reply:
doesn't look like it matters, but i'm not sure yet

Reply:
looks fine

Reply:
It tests fine, and works fine. The issue referenced above does not seem to correlate all that clearly/closely to the described concern above. I've restored the allocated zero value and left a note that this is a haunted graveyard.

Reply:
thanks for the comment!